AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_ModelZoo_Inference_with_FP32_Int8/README.md

```diff
@@ -1,4 +1,4 @@
-# ~Intel Model Zoo` Sample
+# `Intel Model Zoo` Sample
 This code example provides a sample code to run ResNet50 inference on Intel's pretrained FP32 and Int8 model
 
 ## Purpose
```

Comment:
About how users get codes from model zoo, should we direct users to model zoo release inside AI Toolkit installation path instead of model zoo github?
Therefore, users don't need to download model zoo again, and the release is also validated with TF inside AI toolkit.
ex : ` cp -rf /opt/intel/oneapi/modelzoo/latest/models ~/`

Reply:
Hi @louie-tsai after looking into the `README.md` file, I don't see any references to cloning `Model Zoo` and we are telling them in the docs that `Model Zoo` can be found at `/opt/intel/oneapi/modelzoo`.

So I guess no update is needed.

Reply:
@louie-tsai please confirm your ok with the current version or not

Reply:
Indeed, I saw that comments in navigate-to-intel-model-zoo section.
it looks good to me. :)